documentation/src/main/asciidoc/topics/con_discovery.adoc

```diff
@@ -0,0 +1,23 @@
+[id='cluster_discovery-{context}']
+= Cluster Discovery Protocols
+{brandname} supports different protocols that allow nodes to automatically find each other on the network and form clusters.
+
+There are two types of discovery mechanisms that {brandname} can use:
+
+* Generic discovery protocols that work on most networks and do not rely on external services.
+* Discovery protocols that rely on external services to store and retrieve topology information for {brandname} clusters. +
+For instance the JDBC_PING protocol uses a shared database where nodes write their IP addresses.
```

Comment:
Is JDBC_PING a good example here? It seems one of the more niche discovery protocols. How about DNS_PING? That's what we use in kubernetes/openshfit.